src/components/Grid.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { StyleSheet, View } from 'react-primitives';
+import PropTypes from 'prop-types';
+
+import Tile from './Tile';
+
+export default class Grid extends Component {
+    static propTypes = {
+        onPress: PropTypes.func.isRequired,
+        grid: PropTypes.array.isRequired,
+        readOnly: PropTypes.bool,
+    };
+
+    static defaultProps = {
+        readOnly: true,
+    };
+
+    render() {
+        const { grid, readOnly, onPress } = this.props;
+
+        return (
+            <View style={styles.column}>
+                {grid.map((row, rowKey) => {
+                    return (
+                        <View style={styles.row} key={rowKey}>
+                            {row.map((tileValue, tileKey) => {
```

Comment:
Avoid using indexes as key when you can. Here, you should use the `tileValue` as it uniquely identify the tile